qiskit_experiments/library/randomized_benchmarking/standard_rb.py

```diff
@@ -84,6 +84,50 @@ class StandardRB(BaseExperiment, RestlessMixin):
     # section: reference
         .. ref_arxiv:: 1 1009.3639
         .. ref_arxiv:: 2 1109.6887
+
+    # section: example
+        .. jupyter-execute::
+            :hide-code:
+
+            # backend
+            from qiskit_aer import AerSimulator
+            from qiskit_ibm_runtime.fake_provider import FakePerth
+
+            backend = AerSimulator.from_backend(FakePerth())
+
+        .. jupyter-execute::
+
+            import numpy as np
+            from qiskit_experiments.library import StandardRB, InterleavedRB
+            from qiskit_experiments.framework import ParallelExperiment, BatchExperiment
+            import qiskit.circuit.library as circuits
+
+            lengths_2_qubit = np.arange(1, 200, 30)
+            lengths_1_qubit = np.arange(1, 800, 200)
+            num_samples = 10
+            seed = 1010
+            qubits = (1, 2)
+
+            # Run a 1-qubit RB experiment on qubits 1, 2 to determine the error-per-gate of 1-qubit gates
+            single_exps = BatchExperiment(
+                [
+                    StandardRB((qubit,), lengths_1_qubit, num_samples=num_samples, seed=seed)
+                    for qubit in qubits
+                ]
+            )
+            expdata_1q = single_exps.run(backend=backend).block_for_results()
+
+            exp_2q = StandardRB(qubits, lengths_2_qubit, num_samples=num_samples, seed=seed)
+
+            # Use the EPG data of the 1-qubit runs to ensure correct 2-qubit EPG computation
+            exp_2q.analysis.set_options(epg_1_qubit=expdata_1q.analysis_results())
+
+            expdata_2q = exp_2q.run(backend=backend).block_for_results()
+
+            print("Gate error ratio: %s" % expdata_2q.experiment.analysis.options.gate_error_ratio)
+            display(expdata_2q.figure(0))
+            for result in expdata_2q.analysis_results():
+                print(result)
```

Comment:
This (and the interleaved version) also prints out a lot of output. Maybe one result could be printed out to show that there are analysis results but I would not print out all of them. The main results are already included in the legend of the figure so I don't think they need to be printed out at all other than to demonstrate that they are available. Maybe printing just their names is enough?

Reply:
Thank you for your help, @wshanks. I will push a revised version.